src/popup/components/InputField.vue

```diff
@@ -337,11 +352,11 @@ export default defineComponent({
 
   .input-wrapper {
     display: block;
-    padding: 10px 12px; // Decides on the input size
+    padding: 10px 8px 12px 10px; // Decides on the input size
     background-color: var(--color-bg);
     border: none;
     border-radius: $border-radius-interactive;
-    overflow: hidden;
+    position: relative;
```

Comment:
According to commonly used 9elements (https://9elements.com/css-rule-order/) rules the position should be placed at the beginning of the properties.